stake-pool/program/src/state.rs

```diff
@@ -223,7 +223,7 @@ pub struct ValidatorList {
 #[derive(Clone, Copy, Debug, Default, PartialEq, BorshDeserialize, BorshSerialize, BorshSchema)]
 pub struct ValidatorStakeInfo {
     /// Validator vote account address
-    pub vote_account: Pubkey,
+    pub voter_pubkey: Pubkey,
```

Comment:
Why this rename?  This is the vote account, not the voter (authorized voter?).   `vote_account_address` would be most specific, although we use `vote_address` in processor.rs as well.

Reply:
I renamed it to match `Delegation` which uses `voter_pubkey` as the name of the field.

https://github.com/solana-labs/solana/blob/8ec7e2e14f427dc6cfa0243a46091adddfe8d4d2/programs/stake/src/stake_state.rs#L220

Happy to change it to anything else though or revert it! Maybe just back to `vote_account`?

Reply:
Oh yuck.  That's super old terminology.  `voter_pubkey` there ought to be changed as well, again to `vote_account_address` IMO

Reply:
Sounds good, will do!